target_chains/sui/sdk/js/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export { SuiPriceServiceConnection } from "./SuiPriceServiceConnection";
+export { SuiPythClient } from "./client";
```

Comment:
Good to export other useful types as well like [this](https://github.com/pyth-network/pyth-crosschain/blob/main/target_chains/ethereum/sdk/js/src/index.ts#L3C1-L10)